.changeset/dry-avocados-live.md

```diff
@@ -0,0 +1,7 @@
+---
+'docs': patch
```

Comment:
Since we don't publish the docs site we don't need need a changeset for docs updates (can remove this file).